src/main/resources/config.sh

```diff
@@ -0,0 +1,5 @@
+# override the JAVA_HOME value with a known JRE location
+@javaHome@
+
+# use JVM_OPTS to set JVM settings, such as minimum and maximum heap
+#JVM_OPTS="-Xms1g -Xmx1g"
```

Comment:
aha yes - may well not be deploying in a managed way - could just be extracting the generated tgz